core/src/test/java/kafka/test/junit/RaftClusterInvocationContext.java

```diff
@@ -252,7 +220,13 @@ public Admin createAdminClient(Properties configOverrides) {
         public void start() {
```

Comment:
in this method we should always call `format` first. That is a big sugar to users